src/osip-core/src/test/java/edu/kit/pse/osip/core/io/files/ClientSettingsWrapperTest.java

```diff
@@ -53,6 +63,55 @@ public void setUp() throws Exception {
         wrapper = new ClientSettingsWrapper(tempTestFile);
     }
     
+    private void initNullWrapper() throws IOException {
+        tempTestFileNullEntry = File.createTempFile("testClientSettingsTempNullEntry", ".properties");        
+        PrintWriter outStreamNullEntry = new PrintWriter(tempTestFileNullEntry);        
+        outStreamNullEntry.print("fetchIntervall=80");
+        outStreamNullEntry.println();
+        outStreamNullEntry.print("overflowAlarmm_" + (TankSelector.valuesWithoutMix()[0]) + "=" + "false");
```

Comment:
Ich glaube es wäre besser lesbar wenn du TankSelector.valuesWithoutMix()[0] vor den ganzen print Aufrufen in eine Variable speicherst